windows/install_vcpkg.ps1

```diff
@@ -0,0 +1,4 @@
+git clone --depth 1 https://github.com/microsoft/vcpkg
```

Comment:
Can we pin a release to install, instead of cloning from master? For build consistency.